helm-chart/splunk-kubernetes-metrics/templates/deployment.yaml

```diff
@@ -1,29 +1,18 @@
-apiVersion: apps/v1beta1
-kind: Deployment
+apiVersion: extensions/v1beta1
```

Comment:
I'd leave as apps as extensions is being deprecated at some point - https://github.com/kubernetes/kubernetes/issues/43214